lib/init.ts

```diff
@@ -53,6 +53,11 @@ export function init(options: InitParams) {
       `optional cookieDuration is incorrect, expected an integer.`
     );
   }
+  if (this._initialized === true) {
+    throw new Error(
+      `Already initialized. Do not call 'init' method more than once.`
```

Comment:
this is a breaking change, we should probably ignore it or update the parameters instead